autoload/neomake/makers/ft/scss.vim

```diff
@@ -1,12 +1,30 @@
 " vim: ts=4 sw=4 et
 
 function! neomake#makers#ft#scss#EnabledMakers()
-    return ['scsslint']
+    " Only enable first one found
```

Comment:
> Ideally I'd like a `neomake#makers#Prefer([ { sasslint: 'sass-lint' }, { scsslint: 'scss-lint' } ])`

I agree.
OTOH the Python linter is more specific / fine-grained also.

So a method like this might help, but it's not as clear as doing it "inline".